src/home_robot/home_robot/utils/pose.py

```diff
@@ -84,9 +84,12 @@ def to_pos_quat(matrix):
     return pos, np.array([x, y, z, w])
 
 
-def to_matrix(pos, rot) -> np.ndarray:
+def to_matrix(pos, rot, trimesh_format=False) -> np.ndarray:
     """converts pos, quat to matrix format"""
-    x, y, z, w = rot
+    if trimesh_format:
```

Comment:
do we still need trimesh format support?

Reply:
Yes, the changes haven't been ported everywhere. I fear removing this without systematic testing would break our execution code.